### PR TITLE
Fixed up formatting.

### PR DESCRIPTION
**Pull request by finbarr**
Hey folks.  Nice job on releasing the app.  Also, Lockitron is awesome!

This commit fixes up some formatting inconsistencies in your repo.  Removed a bunch of tabs in favor of double-spaces.  Removed a bunch of quadruple spaces in favor of double spaces.  Re-aligned some assignment statements.  Changed a negative conditional to a positive one.

### CODE REVIEW

File: app/views/preorder/share.html.erb [hunk 1]
```diff
@@ -4,23 +4,22 @@
...
+        <%= pin_it_button %>
+      </div>
+      <div id='tweet_button_container'>
+        <%= tweet_button %>
+      </div>
+      <div id='facebook_button_container'>
+        <%= like_button(450, true) %>
+      </div>
+    </div>
+     <div id="order_id">Reservation ID: <%= @order.uuid %>. You can bookmark or print this page for your records.</div>
```
Comment (finbarr) [inline app/views/preorder/share.html.erb:20]: Missed a spot!

**Comment by Jarred-Sumner** [comment]
Finbarr, thanks for taking the time to clean this up! 

I'm sorry for all the formatting inconsistencies, we put this together rather hastily. Clean code is important. If you notice anything else, feel free to send out another pull request.

**Comment by sasasasumna** [comment]
:+1: to what my little bro said

**Comment by finbarr** [comment]
Thanks guys!  Totally get that this was released hurriedly and appreciate it all the more.

**Pull request merged**